modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -61,6 +61,10 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`disableRedirect`
+| disableRedirect controls whether to route all data through the Registry,
```

Comment:
Add backticks to the second `disableRedirect`

s/Registry/registry

Reply:
Others LGTM